File: Java/Mahasiswa.java
public class Mahasiswa extends Human{

    private String nik;
    private String nama;
    private String gender;
    private String prodi;
    private String fakultas;

    Mahasiswa(){
    }

    //getter dan setter
    public void setNik(String nik){
        this.nik = nik;
    }

    public String getNik(){
        return this.nik;
    }

    public void setNama(String nama){
        this.nama = nama;
    }

    public String getNama(){
        return this.nama;
    }

    public void setGender(String gender){
        this.gender = gender;
    }

    public String getGender(){
        return this.gender;
    }

    public void setProdi(String prodi){
		this.prodi = prodi;
	}

	public String getProdi(){
		return this.prodi;
	}

	public void setFakultas(String fakultas){
		this.fakultas = fakultas;
	}

	public String getFakultas(){
		return this.fakultas;
	}

}

File: PHP/Program/Mahasiswa.php
<?php
include "Human.php";

class Mahasiswa extends Human{

	private $nim = '';
	private $prodi = '';
	private $fakultas = '';

	public function __construct($nim = '', $prodi = '', $fakultas = ''){
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

	//getter dan setter
    public function setNim($nim)
	{
		$this->nim = $nim;
	}

	public function getNim()
	{
		return $this->nim;
	}

	public function setProdi($prodi)
	{
		$this->prodi = $prodi;
	}

	public function getProdi()
	{
		return $this->prodi;
	}

	public function setFakultas($fakultas)
	{
		$this->fakultas = $fakultas;
	}

	public function getFakultas()
	{
		return $this->fakultas;
	}

}

?>